basket/news/tasks.py

```diff
@@ -201,6 +202,39 @@ def gmttime():
 
 
 @et_task
+def fxa_delete(data):
+    sfmc.upsert_row('FXA_Deleted', {'FXA_ID': data['uid']})
+
+
+@et_task
+def fxa_verified(data):
+    """Add new FxA users to an SFMC data extension"""
+    # used to be handled by the fxa_register view
+    email = data['email']
+    fxa_id = data['uid']
+    locale = data['locale']
+    subscribe = data.get('marketingOptIn')
+
+    # if we're not using the sandbox ignore testing domains
+    if email_is_testing(email):
+        return
+
+    lang = get_best_language(get_accept_languages(locale))
+    if not lang:
+        return
+
+    _update_fxa_info(email, lang, fxa_id)
+
+    if subscribe:
+        upsert_user.delay(SUBSCRIBE, {
+            'email': email,
+            'lang': lang,
+            'newsletters': settings.FXA_REGISTER_NEWSLETTER,
+            'source_url': settings.FXA_REGISTER_SOURCE_URL,
```

Comment:
@rfk I think this should work well. Do you know if you're currently sending the `source_url` based on the actual FxA domain that sent the event? I'm not sure it matters much for us, but I could do something fancy with this based on whether we're consuming the dev, stage, or prod queue.

Reply:
It's a config value that we *could* change for each environment, but I think right now it just defaults to "accounts.firefox.com" even in dev.  Let me know if you have a use for us changing it and I'll file a bug about updating the config.